Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml

```diff
@@ -387,7 +404,7 @@ tasks:
     task:
       id: 3ccd86fb-693f-405d-81f7-7444add80f10
       version: -1
-      name: True Positive
+      name: Malicious
```

Comment:
Why changing this title?

Reply:
Did some thinking and reading and I think that Malicious and Benign is a better definition than True/False positive since the RDP connection has happened, we just need to decide whether it was malicious or not.